plugins/woocommerce/client/admin/client/settings-payments/settings-payments-main.tsx

```diff
@@ -447,16 +448,17 @@ export const SettingsPaymentsMain = () => {
 	};
 
 	const morePaymentOptionsLink = (
-		<Button
-			variant={ 'link' }
-			target="_blank"
-			href="https://woocommerce.com/product-category/woocommerce-extensions/payment-gateways/"
-			className="more-payment-options-link"
-			onClick={ trackMorePaymentsOptionsClicked }
-		>
-			<img src={ assetUrl + '/icons/external-link.svg' } alt="" />
-			{ __( 'More payment options', 'woocommerce' ) }
-		</Button>
+		<TrackedLink
+			message={ __(
+				// translators: {{Link}} is a placeholder for a html element.
+				'Visit {{Link}}The WooCommerce Marketplace{{/Link}} to find additional payment providers.',
```

Comment:
Shouldn't this be lowercase t? Also, shouldn't we keep up with the section's "Other payment options" naming and say "find additional payment options"?

```suggestion
				'Visit {{Link}}the WooCommerce Marketplace{{/Link}} to find additional payment options.',
```

Reply:
@pmcpinto in the Linear task you provided the new copy as `The WooCommerce Marketplace`. Is the capital `T` an intentional choice?

Whether capitalized or not, I think we should be consistent across all the different placements of this CTA. At the moment, they also use a capital `t`.

Reply:
Updated in ef6f56a